frontend/src/pages/modelServing/ModelServingRoutes.tsx

```diff
@@ -11,11 +11,17 @@ const ModelServingRoutes: React.FC = () => {
 
   return (
     <ProjectsRoutes>
-      <Route path="/" element={<ModelServingContextProvider />}>
+      <Route
+        path="/:namespace?/*"
+        element={
+          <GlobalModelServingCoreLoader
+            getInvalidRedirectPath={(namespace) => `/modelServing/${namespace}`}
+          />
+        }
+      >
```

Comment:
Follow the same way on the pipelines page, make a loader component for the context.